web/react/components/signup_team_complete.jsx

```diff
@@ -496,60 +496,58 @@ SendInivtesPage = React.createClass({
 });
 
 UsernamePage = React.createClass({
-    submitBack: function (e) {
+    submitBack: function(e) {
         e.preventDefault();
-        this.props.state.wizard = "send_invites";
+        this.props.state.wizard = 'send_invites';
         this.props.updateParent(this.props.state);
     },
-    submitNext: function (e) {
+    submitNext: function(e) {
         e.preventDefault();
 
         var name = this.refs.name.getDOMNode().value.trim();
 
         var username_error = utils.isValidUsername(name);
-        if (username_error === "Cannot use a reserved word as a username.") {
-            this.setState({name_error: "This username is reserved, please choose a new one." });
+        if (username_error === 'Cannot use a reserved word as a username.') {
+            this.setState({name_error: 'This username is reserved, please choose a new one.'});
             return;
         } else if (username_error) {
-            this.setState({name_error: "Username must begin with a letter, and contain between 3 to 15 lowercase characters made up of numbers, letters, and the symbols '.', '-' and '_'." });
+            this.setState({name_error: "Username must begin with a letter, and contain between 3 to 15 characters made up of numbers, lowercase letters, and the symbols '.', '-', and '_'"});
```

Comment:
"...characters made up of numbers..."? Is this the text we want here? Characters are not made up of numbers.

How about: "Username must begin with a later, and contain 3 to 15 additional characters that may be numbers, lowercase letters, or any of the symbols '.', '-', or '_'"